features/in_onderzoek.feature

```diff
@@ -280,6 +313,43 @@ Functionaliteit: in onderzoek
       | 050310 | plaats                      | geboorte |
       | 050610 | aangaanHuwelijkPartnerschap | datum    |
 
+      Scenario: persoon heeft gegevens geslacht, gemeenteVanInschrijving en indicatieCurateleRegister in onderzoek
```

Comment:
vanuit testperspectief zou ik hier nog variaties op willen: de categorie heeft wel onderzoek, maar niet het gevraagde gegeven. 
Bijvoorbeeld persooInOnderzoek=010200 (naam). datumIngangOnderzoekPersoon is dan niet opgenomen.
Bijvoorbeeld verblijfplaats.inOnderzoek=011100. datumIngangOnderzoekGemeenten is dan niet opgenomen.